docs/source/glossary.md

```diff
@@ -17,4 +17,22 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
+  
+Bayesian model
+  A Bayesian model is a composite of variables and distributional definitions for these variables. Fundamentally, models are mappings of one set of variables through a probability distribution onto another set of variables. These models define the ways values of some variables can arise, given values of other variables.
+  
+Bayesian data analysis
+  Once we have defined the statistical model, Bayesian data analysis logically processes the data to produce inference. Probability theory is used to represent plausibility, whether in reference to countable events in the world, or theoretical constructs like parameters.
+  
+Likelihood
+  The probability of the data, or the relative number of ways that a value of a parameter can produce the data, is called a likelihood. It is derived by enumerating all the possible data sequences that could have happened and then eliminating those sequences inconsistent with the data.
```

Comment:
The likelihood is sooo important, but there are also many different perspectives.
Best case we could link to both discrete and continuous, univariate and multivariate likelihoods in How-To guides and literature.
I haven't read it, but "Regression and other stories" probably has a chapter about likelihoods.

One resource for likelihood in univariate, continuous scenarios could be our [`calibr8` paper](https://www.biorxiv.org/content/10.1101/2021.06.30.450546v2). I hate to advertise my own work here, but we kinda wrote that thing because the topic wasn't covered from that perspective before.

Reply:
Thank you, re-worded (and minimized).